sim/verilator.wake

```diff
@@ -64,51 +66,75 @@ def compileCmdlineArgs options =
   ++ main
 
 def compileInputs options =
-  def outputDir    = options.getVerilatorCompilePlanOutputDir
-  def includeDirs  = options.getVerilatorCompilePlanIncludeDirs
-  def srcFiles     = options.getVerilatorCompilePlanSourceFiles
-
+  def outputDir = options.getVerilatorCompilePlanOutputDir
+  def includeDirs = options.getVerilatorCompilePlanIncludeDirs
+  def srcFiles = options.getVerilatorCompilePlanSourceFiles
+  def visible = options.getVerilatorCompilePlanVisible
   def includeSources = map (sources _ `.*`) includeDirs | flatten
 
-  source verilatorWrapperScript, mkdir outputDir, includeSources ++ srcFiles# ++ includeDirs
+  source verilatorWrapperScript, mkdir outputDir, includeSources ++ srcFiles ++ visible
 
-global def doVerilatorCompile options =
+def filterVerilatorOutputs all =
+  filter (\f !matches `.*\.(d|dat|tree|dot|a)` f) all
+
+global def runVerilator options =
```

Comment:
Can you add (at least a short, one-sentence) comment describing this function and `doVerilatorCompile`, and specifically describing what is different about them? Within the context of this PR, I understand, but for someone just digging through the codebase, it's less clear what the difference is between the two.